src/ResourceManager/AzureStackAdmin/AzureRM.AzureStackAdmin.psd1

```diff
@@ -9,7 +9,8 @@
 @{
 
 # Version number of this module.
-ModuleVersion = '0.9.11'
+
+ModuleVersion = '0.10.2'
```

Comment:
@bganapa  Can you update the same version in AssemblyInfo.cs also to be consistent